Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m

```diff
@@ -305,6 +300,23 @@ - (void) sendSearchRequestToUpdateTheUsersListWith: (NSString *)filterString {
     
 }
 
+- (NSMutableArray *) manageTheDuplicatedUsers: (NSMutableArray*) items{
```

Comment:
Do you have the method duplicated in two different classes.
Also you are making a for inside other for. It is better just use a dictionary and use the key to know if exist one with that number.